src/Raven.Client/ServerWide/Operations/ServerWideOperation.cs

```diff
@@ -7,7 +7,7 @@ namespace Raven.Client.ServerWide.Operations
     public class ServerWideOperation : Operation
     {
         public ServerWideOperation(RequestExecutor requestExecutor, DocumentConventions conventions, long id, string nodeTag = null)
-            : base(requestExecutor, null, conventions, id)
+            : base(requestExecutor, null, conventions, id, nodeTag)
```

Comment:
Why is the `nodeTag` optional?

Reply:
It shouldn't be, but changing this might cause a compatibility issue.